examples/magento-graphcms/pages/cart.tsx

```diff
@@ -79,29 +81,21 @@ function CartPage() {
         <Container maxWidth='md'>
           <>
             {hasItems ? (
-              <Box sx={(theme) => ({ mt: theme.spacings.lg })}>
+              <>
                 <CartItems
```

Comment:
Maybe we do want to have a bit of spacing on the top, seems a bit cramped on mobile at the moment